subset/bacnet/bacnetTests/src/main/java/FauxDeviceEngine/EntryPoint.java

```diff
@@ -0,0 +1,132 @@
+package FauxDeviceEngine;
+
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.exception.BACnetServiceException;
+import com.serotonin.bacnet4j.npdu.ip.IpNetwork;
+import com.serotonin.bacnet4j.obj.BACnetObject;
+import com.serotonin.bacnet4j.transport.Transport;
+import com.serotonin.bacnet4j.type.enumerated.ObjectType;
+import com.serotonin.bacnet4j.type.enumerated.PropertyIdentifier;
+import com.serotonin.bacnet4j.type.primitive.CharacterString;
+import helper.FileManager;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public class EntryPoint {
+
+    private static int deviceId = 0;
+    private static IpNetwork network;
+    private static LocalDevice localDevice;
+    private static String testCase = "";
+
+    public static void main(String[] args) {
+        if (args.length != 3) {
+            throw new RuntimeException("Usage: testCase localIpAddr broadcastIpAddr");
+        }
+        testCase = args[0];
+        String localIpAddr = args[1];
+        String broadcastIpAddr = args[2];
+        int port = IpNetwork.DEFAULT_PORT;
+        network = new IpNetwork(broadcastIpAddr, port,
+                IpNetwork.DEFAULT_BIND_IP, 0, localIpAddr);
+        Transport transport = new Transport(network);
+        transport.setTimeout(1000);
+
+        try {
+            JSONArray bacnetObjectArray = readJSONFile();
+            getDeviceID(bacnetObjectArray);
+            if(deviceId == 0) {
+                System.out.println("Device ID not found in JSON file. Generating random ID...");
+                deviceId = (int) Math.floor(Math.random() * 1000.0);
+            }
+            System.out.println("Creating LoopDevice id " + deviceId);
+            localDevice = new LocalDevice(deviceId, transport);
+            localDevice.getConfiguration().setProperty(PropertyIdentifier.modelName,
+                    new CharacterString("BACnet4J LoopDevice"));
+            System.out.println("Local device is running with device id " + deviceId);
+            addBacnetProperties(bacnetObjectArray);
+            localDevice.initialize();
+            System.out.println("Device initialized...");
+        } catch (RuntimeException e) {
+            System.out.println("Ex in LoopDevice() ");
+            e.printStackTrace();
+            localDevice.terminate();
+            localDevice = null;
+            throw e;
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static JSONArray readJSONFile() {
+        String jsonFile = "";
+        if (testCase.equals("pass")) {
+            jsonFile = "Faux-Device-Pass.json";
```

Comment:
Can we pass these json filenames as arguments somewhere? You might find that the filename changes in the future and you're having to unnecessarily rebuild docker containers...